releases/release-6.5.7.md

```diff
@@ -0,0 +1,151 @@
+---
+title: TiDB 6.5.7 Release Notes
+summary: 了解 TiDB 6.5.7 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 6.5.7 Release Notes
+
+发版日期：2023 年 x 月 x 日
+
+TiDB 版本：6.5.7
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.5.7#version-list)
+
+## 兼容性变更
+
+无
+
+## 改进提升
+
++ TiDB
+    <!-- tw@Oreoxmt -->
+    - (dup): release-7.4.0.md > 改进提升> TiDB - 优化 `ANALYZE` 分区表的内存使用和性能 [#47071](https://github.com/pingcap/tidb/issues/47071) [#47104](https://github.com/pingcap/tidb/issues/47104) [#46804](https://github.com/pingcap/tidb/issues/46804) @[hawkingrei](https://github.com/hawkingrei)
+    - Plan Cache 支持主键的 `IN (....)` 语法[#44830](https://github.com/pingcap/tidb/issues/44830) @[qw4990](https://github.com/qw4990)
+    - 提供 TiDB 等待统计信息加载完成才开始对外提供服务的开关 [#43385](https://github.com/pingcap/tidb/issues/43385) @[xuyifangreeneyes](https://github.com/xuyifangreeneyes)
+    - 增强特定情况 OUTER JOIN 转 INNER JOIN 的能力 [#49616](https://github.com/pingcap/tidb/issues/49616) @[qw4990](https://github.com/qw4990)
+
++ TiKV
+
+    - note [#issue](https://github.com/tikv/tikv/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+    - note [#issue](https://github.com/tikv/tikv/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+
++ PD
+
+    - note [#issue](https://github.com/tikv/pd/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+    - note [#issue](https://github.com/tikv/pd/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+
++ TiFlash
+
+    <!-- tw@Oreoxmt -->
+    - 减少磁盘性能抖动对读取延迟的影响 [#8583](https://github.com/pingcap/tiflash/issues/8583) @[JaySon-Huang](https://github.com/JaySon-Huang)
+    - 日志等级默认调整为 "info" 减少日志打印的开销 [#8568](https://github.com/pingcap/tiflash/issues/8568) @[xzhangxian1008](https://github.com/xzhangxian1008)
```

Comment:
@xzhangxian1008 This enhancement seems to impact the TiFlash configuration document. Could you please create PRs for both docs-cn and docs repositories to update the default value of `logger.level`, along with any other related documentation?

https://github.com/pingcap/docs-cn/blob/1b4cf206c7158c97be4f3a0738034b620abd0215/tiflash/tiflash-configuration.md?plain=1#L164-L166